code/modules/client/preferences_savefile.dm

```diff
@@ -69,6 +69,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		S.dir.Remove("overhead_chat")
 	if(current_version < 35)
 		see_balloon_alerts = BALLOON_ALERT_ALWAYS
+	if(current_version < 36)
+		WRITE_FILE(S["helmet_style"], HELMET_DEFAULT)
```

Comment:
Don't burn a savefile version, your sanity check should default it to `HELMET_DEFAULT` anyways